gtdynamics/utils/Initializer.h

```diff
@@ -0,0 +1,223 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file  initialize_solution_utils.h
+ * @brief Utility methods for initializing trajectory optimization solutions.
+ * @author: Alejandro Escontrela, Yetong Zhang
+ */
+
+#pragma once
+
+#include <gtdynamics/dynamics/DynamicsGraph.h>
+#include <gtdynamics/universal_robot/Robot.h>
+#include <gtsam/base/Value.h>
+#include <gtsam/base/Vector.h>
+#include <gtsam/geometry/Pose3.h>
+#include <gtsam/linear/Sampler.h>
+#include <gtsam/slam/PriorFactor.h>
+
+#include <boost/optional.hpp>
+#include <random>
+#include <string>
+#include <vector>
+
+namespace gtdynamics {
+
+class Initializer {
+
+ public:
+    
+    // Default Constructor
+    Initializer() {}
+
+    /**
+     * Add zero-mean gaussian noise to a Pose3.
+     *
+     * @param T Pose3 which to add noise to.
+     * @param sampler Helper to sample values from a gaussian distribution.
+     */
+    gtsam::Pose3 AddGaussianNoiseToPose(const gtsam::Pose3& T,
```

Comment:
Apologies for bringing this up so late, but shouldn't the method names start with lower case if they are not static methods? This is to be consistent with the rest of the codebase (even though it is not exactly inline with the Google Style).

The [ISO C++ Standard](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#S-naming) is a better guide IMO.